projects/alpaca-finance-2.0/index.js

```diff
@@ -0,0 +1,62 @@
+const sdk = require("@defillama/sdk");
+const abi = require("./abi.json");
+const { getCache, getConfig } = require("../helper/cache");
+const BigNumber = require("bignumber.js");
+
+async function lendingTvl(timestamp, _, _1, {api}) {
```

Comment:
Should we keep index file be simpler to make it easy to maintain when new service available on v2?
Ref. from v1, we can move these functions to new file like `moneyMarket.js`